package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -113,24 +113,31 @@
 
 
 def wc_pair(universe, i, bp, seg1="SYSTEM", seg2="SYSTEM"):
-    """Watson-Crick basepair distance for residue *i* with residue *bp*.
+    """Watson-Crick basepair distance for residue `i` with residue `bp`.
 
     The distance of the nitrogen atoms in a Watson-Crick hydrogen bond is
     computed.
 
-    :Arguments:
-      *universe*
-          :class:`~MDAnalysis.core.universe.Universe` containing the trajectory
-      *seg1*
-          segment id for first base
-      *i*
-          resid of the first base
-      *seg2*
-          segment id for second base
-      *bp*
-          resid of the second base
+    Parameters
+    ----------
+    universe : Universe
+         :class:`~MDAnalysis.core.universe.Universe` containing the trajectory
+    i : int
+        resid of the first base
+    bp : int
+        resid of the second base
+    seg1 : str (optional)
+        segment id for first base ["SYSTEM"]
+    seg2 : str (optional)
+        segment id for second base ["SYSTEM"]
+
+    Returns
+    -------
+    float
+        Watson-Crick base pair distance
+
 
-    .. NOTE:: If failure occurs be sure to check the segment identification.
+    .. Note:: If failure occurs be sure to check the segment identification.
```

Comment:
Changed to Notes section (rebased so it now shows up in commit 056f0976100e02c82a354a76be79f0669549926e )